### PR TITLE
Improve tooltip styling

### PR DESCRIPTION
**Pull request by martinRenou**
Old visual aspect:

old_tooltip

New visual aspect:

new_tooltip_font

### CODE REVIEW

File: js/less/bqplot.less [hunk 1]
```diff
@@ -50,7 +50,7 @@
     --bq-selector-fill: var(--jp-border-color1);
 
     --bq-mark-tooltip-border: 2px solid var(--jp-border-color0);
-    --bq-mark-tooltip-box-shadow: 1px 1px 2px rgba(245, 245, 245, 0.8);
+    --bq-mark-tooltip-box-shadow: var(--jp-elevation-z6);
```
Comment (martinRenou) [inline js/less/bqplot.less:53]: This seems to be the value used in JupyterLab for box shadows

Comment (martinRenou) [reply js/less/bqplot.less]: Actually this variable is not there in Jupyter Notebook, so I now use the actual variable value.

**Comment by maartenbreddels** [comment]
Really nice, about 10x nicer in my opinion :)

**Comment by martinRenou** [comment]
It would also be nice if the tooltip font style had the same default values as the axis font one.

**Comment by SylvainCorlay** [comment]
Could you please add a screenshot with the new font?

**Comment by martinRenou** [comment]
I updated the PR comment with a new screenshot

**Comment by SylvainCorlay** [comment]
This looks *soo* much better.

**Pull request merged**